jiwonjang/app.js

```diff
@@ -47,9 +49,124 @@ app.post("/users", async (req, res) => {
       password
     ) VALUES (?, ?, ?, ?);
     `,
-    [name, email, profileImage, password]
+    [name, email, profile_image, password]
+  );
+  res.status(201).json({ message: "userCreated" });
+});
+
+app.post("/posts", async (req, res) => {
+  const { title, content, user_id, posts_img } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+      title,
+      content,
+      user_id,
+      posts_img
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [title, content, user_id, posts_img]
   );
-  return res.status(201).json({ message: "useCreated" });
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.patch("/posts", async (req, res) => {
+  const { posting_id, posting_title, posting_content } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+      SET
+      title = ?,
+      content = ?
+    WHERE id = ?
+    `,
+    [posting_title, posting_content, posting_id]
+  );
+  const result = await appDataSource.query(
+    `SELECT 
+        users.id as userId,
+        users.name as userName,
+        posts.id as postingId,
+        posts.title as postingTitle,
+        posts.content as postingContent
+      FROM posts
+      INNER JOIN users ON users.id = posts.user_id
+      WHERE posts.id = ?
+      GROUP BY users.id
+    `,
+    [posting_id]
+  );
+  return res.status(200).json({ data: result });
+});
+
+app.delete("/:postId/posts", async (req, res) => {
```

Comment:
```suggestion
app.delete("/posts/:postId", async (req, res) => {
```
"포스트들 중, 특정 id값 하나"가 엔드포인트를 읽을 때 보다 자연스러울 것 같습니다.